css/grids.css

```diff
@@ -19,36 +18,38 @@ heading2{
 	padding-left: 5%;
 	width: 100%;
 	display: grid;
-	grid-gap: 1%;
-	grid-template-columns: 2% 11% 11% 11% 11% 11% 11% 11% 11%;
-	grid-template-rows: 80px;	
+	grid-gap: 2%;
+	grid-template-columns: 0% 9% 11% 11% 11% 11% 11% 11% 11% 11%;
+	grid-template-rows: 70px;	
 	
 	
 }
 
+
+
+	
 /* colour rows for queries (each song)*/
+
 heading2:nth-child(even) {
-	background: #404040;
-	/*opacity: 50%; */
+	background: #dd9a4f;
```

Comment:
added the same background to the column to create contrast within the other query column; orange colour